k8s/manifest-templates/pharos.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pharos
+  labels:
+    name: pharos
+spec:
+  hostNetwork: true
+  containers:
+  - name: pharos-dashboard
+    image: ereslibre/pharos-dev:latest
```

Comment:
Yes, that's why I suggest using `pharos:development` so the kubelet won't try to fetch it every time (that only happens with the `latest` tag). With `imagePullPolicy: Never` we would be lacking any kind of pulling (but we want to use an image on the registry).